module/module.go

```diff
@@ -31,17 +32,50 @@ import (
 	rutils "go.viam.com/rdk/utils"
 )
 
-// CheckSocketAddressLength returns an error if the socket path is too long for the OS.
-func CheckSocketAddressLength(addr string) error {
+const (
+	socketSuffix = ".sock"
+	// If we assume each robot has 10 modules running on it, and they would all have colliding truncated names,
+	// P(collision) = 1-(58^5)!/((58^5-10)!*(58^(5*10))) ~ 6.8E-8.
+	socketRandomSuffixLen int = 5
+	base58Chars               = "123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz"
```

Comment:
I am not sure this belongs here. Unfortunately, it is nowhere else in our codebase.


Happy to move this to a utils file or to rename it to `socketRandomSuffixAlphabet`